lib/playgroundbook_lint/abstract_linter.rb

```diff
@@ -1,7 +1,7 @@
 require 'colored'
 require 'cork'
 
-module PlaygroundBookLint
+module Playgroundbook
```

Comment:
I'm curious what the rationale behind the change from `PlaygroundBook` to `Playgroundbook` is